### PR TITLE
Correct and test display output for IntervalBox with StaticArrays

### PR DESCRIPTION
**Pull request by dpsanders**

### CODE REVIEW

File: src/multidim/intervalbox.jl [hunk 1]
```diff
@@ -9,7 +9,7 @@ immutable IntervalBox{N,T} <: StaticVector{Interval{T}}
     data::NTuple{N,Interval{T}}
 end
 
-IntervalBox{N,T}(x::NTuple{N,Interval{T}}) = IntervalBox{N,T}(x)
+# IntervalBox{N,T}(x::NTuple{N,Interval{T}}) = IntervalBox{N,T}(x)
```
Comment (lbenet) [inline src/multidim/intervalbox.jl]: Any reason for commenting this test?

**Comment by dpsanders** [comment]
It's a constructor, not a test. Apparently it overwrites the default
constructor and is not required.

**Comment by lbenet** [comment]
You are right, sorry. I think this is also ready. Shall I merge?

**Comment by dpsanders** [comment]
Done!

**Pull request merged**